specification/consumption/resource-manager/Microsoft.Consumption/stable/2019-10-01/consumption.json

```diff
@@ -1829,616 +1829,6 @@
         }
       }
     },
-    "LegacyUsageDetail": {
```

Comment:
>"LegacyUsageDetail" [](start = 4, length = 19)

You are updating the contract of an existing api version, you either need to get a breaking change exception or move to a new api.